### ISSUE TITLE
Debug code not removed

### ISSUE THREAD

**Issue by DinuruSeniya** [rcmaehl/MSEdgeRedirect#153]
### Preflight Checklist

- [X] Microsoft Edge is still installed (see issue 26)
- [X] Running microsoft-edge:https://google.com redirects successfully
- [X] Microsoft Edge is selected in any "How do you want to open this?" box (if applicable)

### Install Type

Manual Update

### Install Mode

Active Mode

### Steps to reproduce

Update to version 0.7.0.0
MSN News redirect to Google should be selected when upgrading
Select a news article from the Widgets menu (Win + W).

### ✔️ Expected Behavior

The user-defined browser should open with the redirected link.

### ❌ Actual Behavior

![image](https://user-images.githubusercontent.com/48905657/177856561-ac49053a-f86d-4ac8-851f-81f9a54afedd.png)
This debug box shows up. After clicking ok, the user-defined browser opens with the redirected link.

### Microsoft Windows version

22000.778 (Windows 11 21H2)

### Other Software

None

**Comment by rcmaehl** [comment]
Yep. Was fixed in https://github.com/rcmaehl/MSEdgeRedirect/commit/0a258b2694108a7939ce6c923dad1c02dcf2e093

Will be fixed in 0.7.1.0 in the next couple days

**Comment by DinuruSeniya** [comment]
Installed the nightly build, now the box is gone but now all links point to https://www.google.com/url?q=https://www.merriam-webster.com/dictionary/other

**Comment by rcmaehl** [comment]
> Installed the nightly build, now the box is gone but now the 'other' links all point to https://www.google.com/url?q=https://www.merriam-webster.com/dictionary/other @rcmaehl

Yep. News redirects are a bit alpha. I'm working on improving them now.
